apps/next-frontend/.eslintrc.js

```diff
@@ -0,0 +1,200 @@
+/** @type {import("eslint").Linter.Config} */
```

Comment:
Do we need all this for nextJs?

Reply:
it was there previously so i thought of keeping it as it is